app/assets/javascripts/home.js

```diff
@@ -0,0 +1,68 @@
+
+$(document).ready(function(){
+ var petList = $("#recent-pet-list");
+ getPets();
+ });
+
+ function getPets(){
+   $.ajax({
+     url:`api/v1/pets`,
+     success: function(pets){
+       console.log(pets[0])
+       renderPets(pets);
+     },
+     error: function(){
+       alert('Could not get pet');
+     }
+   })
+ }
+
+function renderPets(pets){
+  var petTemplate = $('#recent-pet-list').html();
+  var petList = $('#recent-pet-list');
+
+  var petsHTML = pets.slice(0,5).map(function(pet){
+    return PetListItem(pet);
+  }).join("");
+
+  petList.html(petsHTML);
+}
+
+function PetListItem (pet) {
+  return `
+  <div class="main-list">
+    <div class="col-md-3 col-xs-6">
+      <img class='img-circle' src="${pet.image}" alt="" />
```

Comment:
It works! In the report it shows 3 images, and in the Lost pet scrollbar it just shows the first one :)